presto-main/src/main/java/com/facebook/presto/execution/resourceGroups/InternalResourceGroupManager.java

```diff
@@ -473,4 +504,20 @@ private static <C> ResourceGroupConfigurationManager<C> cast(ResourceGroupConfig
     {
         return (ResourceGroupConfigurationManager<C>) manager;
     }
+
+    private static class InitializingConfigurationManager
+            implements ResourceGroupConfigurationManager
```

Comment:
nit : add generics to satisfy compiler
ResourceGroupConfigurationManager\<VoidContext> / ResourceGroupConfigurationManager\<Void>